tests/tests/cpld_bmarkfrac_v16_35d

```diff
@@ -0,0 +1,95 @@
+#
+#  cpld_bmark_v16 35D
+#
+export TEST_DESCR="Fully coupled FV3-CCPP-MOM6-CICE-CMEPS system - C384L127 MX025 - Benchmark 35d test with v16"
+
+export_35d_run
+export_fv3
+export_cpl
+
+export SYEAR
+export SMONTH
+export SDAY="01"
+export SHOUR="00"
+
+export DAYS="35"
+export FHMAX="840"
+export RESTART_N=${FHMAX}
+export WLCLK=480
+
+export TASKS=$TASKS_cpl_bmrk
+export TPN=$TPN_cpl_bmrk
+export INPES=$INPES_cpl_bmrk
+export JNPES=$JNPES_cpl_bmrk
+export THRD=$THRD_cpl_bmrk
+export WRTTASK_PER_GROUP=$WPG_cpl_bmrk
```

Comment:
For the cpld_bmarkfrac_v16_35d and the cpld_restart_bmarkfrac_v16 test, do you want to also turn on inline post by adding "export WRITE_DOPOST=.true."?